src/components/CreateResolution/CompleteResolutionSummary.vue

```diff
@@ -129,6 +137,11 @@ export default class CompleteResolutionSummary extends Mixins(DateMixin) {
     return '(Not Entered)'
   }
 
+  @Watch('getCreateResolutionStep.resolutionText')
+  private onResolutionTextChanged (): void {
```

Comment:
Can you just watch `this.resolutionText`, instead of the field on the getter?

Reply:
Or could you add a method to the `@change` event of resolutionTextRef?

Reply:
true, let me test that.

Reply:
updated to watch `resolutionText`.  Tried `@change` but things weren't updating properly.